src/index.scss

```diff
@@ -227,7 +227,7 @@ header {
 
 .header-action-bar {
   background-color: #fff;
-  z-index: 2;
+  z-index: 9999;
```

Comment:
Is there any need for changing z-index?

Reply:
Fix action menu modal and header overlapping